theme/page.template

```diff
@@ -15,7 +15,7 @@
 <![endif]-->
 <link rel="shortcut icon" href="">
 </head>
-<body>
+<body dir="<%= direction %>">
```

Comment:
Yes, This type setting is true and complete, but in script mode must be left to right that i change app.css for this result.
I am writing new book with this new change in Persian language,